modules/apps/client-extension/client-extension-api/src/main/java/com/liferay/client/extension/constants/ClientExtensionEntryConstants.java

```diff
@@ -12,6 +12,9 @@ public class ClientExtensionEntryConstants {
 
 	public static final String TYPE_CUSTOM_ELEMENT = "customElement";
 
+	public static final String TYPE_EDITOR_CONFIGURATION =
+		"editorConfiguration";
```

Comment:
@dsanz Is it intentional that we are using `editor` everywhere and not `ckeditor4`? Like here, shouldn't it be `TYPE_CKEDITOR_4_CONFIGURATION = "ckeditor4Configuration"`? Is the intent that this CX covers other editors in DXP, like Ace editor? If so, should we update labels in mockups?